tasks.py

```diff
@@ -0,0 +1,289 @@
+    if debug:
+        getLogger().setLevel("DEBUG")
+
+    try:
+        subprocess.run(
+            [
+                "pipenv",
+                "run",
+                "pytest",
+                "--cov=release_monitor",
+                "tests",
+            ],
+            capture_output=True,
+            check=True,
+        )
+    except subprocess.CalledProcessError as error:
+        LOG.error(
+            f"Testing failed with stdout of {error.stdout.decode('utf-8')} and stderr of {error.stderr.decode('utf-8')}"
+        )
+        sys.exit(1)
+
+
+@task
+def reformat(_c, debug=False):
+    """Reformat release_monitor"""
+    if debug:
+        getLogger().setLevel("DEBUG")
+
+    entrypoint_and_command = [
+        ("isort", ". --settings-file /action/lib/.automation/.isort.cfg"),
+        ("black", "."),
+    ]
+    image = "seiso/goat:latest"
+    working_dir = "/goat/"
+    volumes = {CWD: {"bind": working_dir, "mode": "rw"}}
+
+    LOG.info("Pulling %s...", image)
+    CLIENT.images.pull(image)
+    LOG.info("Reformatting the project...")
+    for entrypoint, command in entrypoint_and_command:
+        container = CLIENT.containers.run(
+            auto_remove=False,
+            command=command,
+            detach=True,
+            entrypoint=entrypoint,
+            image=image,
+            volumes=volumes,
+            working_dir=working_dir,
+        )
+        process_container(container=container)
+
+
+@task
+def update(_c, debug=False):
+    """Update the core components of release_monitor"""
+    if debug:
+        getLogger().setLevel("DEBUG")
+
+    # Update the CI dependencies
+    image = "python:3.9"
+    working_dir = "/usr/src/app/"
+    volumes = {CWD: {"bind": working_dir, "mode": "rw"}}
+    CLIENT.images.pull(repository=image)
+    command = '/bin/bash -c "python3 -m pip install --upgrade pipenv &>/dev/null && pipenv update"'
+    container = CLIENT.containers.run(
+        auto_remove=False,
+        command=command,
+        detach=True,
+        image=image,
+        volumes=volumes,
+        working_dir=working_dir,
+    )
+    process_container(container=container)
+
+
+@task
+def release(_c, debug=False):
+    """Make a new release of release_monitor"""
+    if debug:
+        getLogger().setLevel("DEBUG")
+
+    if REPO.head.is_detached:
+        LOG.error("In detached HEAD state, refusing to release")
+        sys.exit(1)
+
+    # Get the current date info
+    date_info = datetime.now().strftime("%Y.%m")
+
+    # Our CalVer pattern which works until year 2200, up to 100 releases a
+    # month (purposefully excludes builds)
+    pattern = re.compile(r"v2[0-1][0-9]{2}.(0[0-9]|1[0-2]).[0-9]{2}")
+
+    # Identify and set the increment
+    for tag in reversed(REPO.tags):
+        if pattern.fullmatch(tag.name):
+            latest_release = tag.name
+            break
+    else:
+        latest_release = None
+
+    if latest_release and date_info == latest_release[:7]:
```

Comment:
This is using an old version of this with a bug. 

I suggest regenerating

https://github.com/SeisoLLC/cookiecutter-python/blob/eec275779cd02c083c308644e58bd532e75c8fd5/%7B%7Bcookiecutter.project_slug%7D%7D/tasks.py#L263

Reply:
This took a while to regenerate on python 3.9. It is now the latest rendered cc-python slug.